docs/en/classic/compute-admin-guide/configure/permissions.adoc

```diff
@@ -12,7 +12,7 @@ Review the following sets of permissions to ensure Prisma Cloud has the right pe
 ** <<#azure,Azure required permissions>>
 ** <<#gcp,GCP required permissions>>
 
-* The xref:#min-permission-cloud-discovery[minimal permissions needed for cloud discovery].
+* <<#min-permission-cloud-discovery>>.
```

Comment:
This change is ok but let's try to use xrefs consistently instead. This in-topic linking works but it is very troublesome to change when content is restructured.